src/main/java/com/gd/clinic/entity/Patient.java

```diff
@@ -0,0 +1,75 @@
+package com.gd.clinic.entity;
+
+import com.gd.clinic.model.NewPatientDto;
+import lombok.*;
+import org.hibernate.Hibernate;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.time.OffsetDateTime;
+import java.util.*;
+
+
+@RequiredArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "patients")
+public class Patient {
+
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
+    @Type(type = "pg-uuid")
+    private UUID id;
+
+    @NonNull
+    private String firstName;
+
+    @NonNull
+    private  String lastName;
+
+    @NonNull
+    private String diagnosis;
+
+    @NonNull
+    private String insuranceNumber;
+
+    @NonNull
+    private OffsetDateTime birthDate;
+
+    @Type(type = "pg-uuid")
+    //TODO: Uncomment when branch "security" is merged or vice versa to reference the user entity for the doctor id
```

Comment:
I guess security branch has already been merged, so you can fix this

Reply:
Indeed, all commented code has been integrated this commit.